Makefile

```diff
@@ -21,11 +33,17 @@ lint: install-tools
 compile:
 	# this is a workaround to compile and cache all of the tests without actually running any of them
 	go test -run=NO_MATCH ./...
+	# Also build the load generator programs for each os/platform as a sanity check.
+	$(LOADGEN_WIN_BUILD)/ github.com/aws/amazon-cloudwatch-agent-test/cmd/...
+	$(LOADGEN_LINUX_AMD64_BUILD)/ github.com/aws/amazon-cloudwatch-agent-test/cmd/...
+	$(LOADGEN_LINUX_ARM64_BUILD)/ github.com/aws/amazon-cloudwatch-agent-test/cmd/...
+	$(LOADGEN_DARWIN_AMD64_BUILD)/ github.com/aws/amazon-cloudwatch-agent-test/cmd/...
+	$(LOADGEN_DARWIN_ARM64_BUILD)/ github.com/aws/amazon-cloudwatch-agent-test/cmd/...
```

Comment:
This will compile _everything_ in the `cmd` directory for each of the OSes.
This just helps devs do a quick sanity check that the stuff they are about to commit at least compiles...